integration/cursors/awaitdata_test.go

```diff
@@ -308,6 +308,122 @@ func TestCursorsTailableAwaitData(t *testing.T) {
 	})
 }
 
+func TestCursorsTailableAwaitDataAfterInsertStress(t *testing.T) {
+	if !setup.IsMongoDB(t) {
+		t.Skip("https://github.com/FerretDB/FerretDB/pull/3961")
```

Comment:
We should link to issues, not to PRs.

And can we use expected failure instead?

Reply:
Most of the subtests pass, and usually one of them fails, so not really